test/e2e/policies.test-suite/balloons/n4c16/test22-isolcpus/code.var.sh

```diff
@@ -0,0 +1,33 @@
+helm-terminate
+helm_config=${TEST_DIR}/balloons-isolcpus.cfg helm-launch balloons
+
+vm-command "grep isolcpus=0,1 /proc/cmdline" || {
+    if [ "$distro" = "generic/fedora37" ]; then
```

Comment:
Let's not assume it's fedora37 here, let's just check for fedora.

Reply:
Fixed.